### ISSUE TITLE
Shift stopped working on new Android versions

### ISSUE THREAD

**Issue by acassis** [boombuler/Shift#2]
Shift game is not working on new Android version (i.e. Android 7.1.1 on Moto Z Play). Some friends also reported the same issue on different smartphones.

**Comment by boombuler** [comment]
Yes indeed. The problem here is that cocos 2d (game engine) version doesn't work with newer android versions... to get it working I would need a general rewrite :(

**Comment by acassis** [comment]
Hi @boombuler I missed your comment, sorry! Maybe it could be fixed for new cocos lib, right? Anyway I created by own version in hardware (physical version I mean) running on NuttX RTOS: https://www.youtube.com/shorts/dsUN4EQ0_JI
